koku/masu/test/external/downloader/aws/test_aws_report_downloader.py

```diff
@@ -733,8 +733,10 @@ def test_create_daily_archives_check_leading_zeros(self, mock_copy):
 
         os.remove(temp_path)
 
-    def test_create_daily_archives_dates_out_of_range(self):
+    @patch("masu.util.aws.common.get_s3_objects_not_matching_metadata")
+    def test_create_daily_archives_dates_out_of_range(self, mock_get_s3_objects):
         """Test that we correctly create daily archive files."""
+        mock_get_s3_objects.return_value = []
```

Comment:
A simpler solution to fixing this test is to flip the `setup_complete` to False. This test is relying on this pathway:
https://github.com/project-koku/koku/blob/65fedd11b84e4e90677acc1efe2a1b0efb3b8c31/koku/masu/external/downloader/aws/aws_report_downloader.py#L84